rskj-core/src/main/java/co/rsk/rpc/netty/JsonRpcWeb3ServerHandlerInvocationListener.java

```diff
@@ -0,0 +1,54 @@
+package co.rsk.rpc.netty;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.googlecode.jsonrpc4j.InvocationListener;
+
+import java.lang.reflect.Method;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.ethereum.rpc.exception.RskJsonRpcRequestException.invalidParamError;
+
+public class JsonRpcWeb3ServerHandlerInvocationListener implements InvocationListener {
+    private final Map<String, Boolean> methodsToCheckMap = new HashMap<>();
```

Comment:
What it is the intention of this map? It seems that all the items has the value set to true.

Reply:
My reasoning here is that there may be methods that do not require a check for its argument's data types, this map constains the methods that should be checked, so far I added the one I worked on for this ticket.